### PR TITLE
knoxite: init at 2020-06-15

### PR DESCRIPTION
**Pull request by penguwin**
###### Motivation for this change

[knoxite](https://www.knoxite.com) is a secure data storage and backup system.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/backup/knoxite/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    repo = pname;
+    rev = "bfdf2fab2325fb2e7521fdf32bbb6edbab4c9896";
+    sha256 = "0fn0rw7xs1q3ybdcbnwwk61wn326n64085gv6qyrn0267vjrnhwj";
+  };
+
+  vendorSha256 = "0cwz0ml384zyq85rln8yi8ymdbc6xb7zy3r2yg135js6r3jjripf";
+
+  meta = with stdenv.lib; {
+    description     = "A secure data storage and backup system";
+    homepage        = "https://www.knoxite.com";
```
Comment (doronbehar) [inline pkgs/tools/backup/knoxite/default.nix:18]: That's 404.

File: pkgs/tools/backup/knoxite/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "knoxite";
+  version = "git-2020-06-15";
```
Comment (doronbehar) [inline pkgs/tools/backup/knoxite/default.nix:5]:
```suggestion
  version = "unstable-2020-06-15";
```

Comment (penguwin) [reply pkgs/tools/backup/knoxite/default.nix]: Applied change as requested

File: pkgs/tools/backup/knoxite/default.nix (hunk 1 above)
Comment (penguwin) [reply pkgs/tools/backup/knoxite/default.nix:18]: There was an [issue](https://github.com/containous/traefik/pull/7039) with our traefik which is fixed now.

**Review by doronbehar — APPROVED** [review]
Passes `nixpkgs-review` build and `--help` test.

**Comment by penguwin** [comment]
If there's anything I can help you with please let me know

**Comment by penguwin** [comment]
Converting to draft, gonna reopen with the next stable release of knoxite

**Comment by doronbehar** [comment]
No official release of knoxite yet, so it seems..

**Comment by doronbehar** [comment]
@penguwin it's OK to add a package without releases, using the `unstable-YYYY-MM_DD`, see [documentation](https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming).

**Pull request closed without merging**